sdcm/cluster.py

```diff
@@ -1427,10 +1427,10 @@ def _read_system_log_and_publish_events(self,
                             item.period_func(match=event_match)
 
                     # for each line use all regexes to match, and if found send an event
-                    for pattern, event in SYSTEM_ERROR_EVENTS_PATTERNS:
+                    for pattern, event in SYSTEM_ERROR_EVENTS_PATTERNS + INSTANCE_STATUS_EVENTS_PATTERNS:
```

Comment:
We can extract this to a variable somewhere outside of the loop, to avoid repeating this operation on every iteration.

Reply:
done